tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "moduleResolution": "node",
-    "target": "es6",
-    "module": "es2015",
-    "lib": ["es2015", "es2016", "es2017", "dom"],
+    "target": "esnext",
```

Comment:
Would we still be able to support IE 11? The goal is that everywhere WebGL is supported the library should be able to run.

Reply:
> Great! I think that would work well 👍
> 
> What would the `mobileTiers` and `desktopTiers` now signify?

Right now it is using the same fps values as in the test it pulls the data from.

Reply:
Yeah, it would be good to have a minified polyfilled version and a bare version, I guess.. That way when you are consuming it for use in a web app  using webpack for example, it can use its own browserlist / polyfill setup etc.

Reply:
I have been looking into this more, and while I was expecting rollup to offer a backwards compatible option to load chunks using a combination of Fetch/eval it does not seem to do so.. Another option would be to move the `json` files to a public directory and load them using fetch/xmlrequest